analytical/templatetags/chartbeat.py

```diff
@@ -57,6 +56,8 @@ def chartbeat_top(parser, token):
 
 class ChartbeatTopNode(Node):
     def render(self, context):
+        if settings.get("DISABLE_TRACKING_CODE", False):
+            return ""
```

Comment:
This code look like being repeated over and over again throughout all analytical modules. Can you put this in a parent class or mixin, instead?